description.md

```diff
@@ -34,6 +34,50 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Practical Sessions
 
+Practical sessions are represented by hands-on activities that learners perform in order to acquire practical experience.
+These activities are typically supervized by a teacher that proposes the practical exercices and evaluates the learner's performance.
+The purpose of the evaluation is mainly to provide constructive feedback, as opposed to grading the student.
+
+Instances of practical sessions are represented by laboratories, seminars or workshops.
+The format of a practical session may include:
+
+- theoretical and practical explanations offered by the teacher.
```

Comment:
The first item should be "support material" in the form of reading content and media, and slides.

Reply:
No, this is not a description of the materials needed for practical sessions, but rather a description of the high-level activities that take place during a practical session. What you are asking for is treated below.